scripts/ci.go

```diff
@@ -0,0 +1,71 @@
+// +build none
+
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"runtime"
+	"strings"
+)
+
+const (
+	GOLANGCI_VERSION = "github.com/golangci/golangci-lint/cmd/golangci-lint@v1.23.1"
```

Comment:
this is prune to deprecation, is there a better alternative?

Reply:
This is ok, using latest version on this repo is a bad idea. For ex, latest version v24 breaks with some weird errors. It is a good idea to force a version and having us deciding when to update it manually.